src/styles/overrides/_se-form.scss

```diff
@@ -0,0 +1,92 @@
+.seform {
+  form {
+    .seText label {
+      @extend .dg_label;
+    }
+
+    fieldset {
+      @extend .dg_fieldset;
+
+      legend {
+        @extend .dg_legend;
+      }
+
+      .seRadioLabel {
+        @extend .dg_label;
+        @extend .dg_radio-label;
+      }
+
+      .seCheckboxLabel {
+        @extend .dg_label;
+        @extend .dg_checkbox-label;
+      }
+
+      .seRadio {
+        @extend .dg_radio-input;
+      }
+
+      .seCheckbox {
+        @extend .dg_checkbox-input;
+      }
+
+      .seLabelCell {
+        @extend .dg_label;
+        margin-top: 0;
+        margin-bottom: 5px;
+
+        .seRequiredMarker {
+          margin-left: 5px;
```

Comment:
Should this be set to the variable `$base-spacing-unit`? Same with line 35.

Reply:
yes

Reply:
this has been addressed.